app/javascript/src/components/form/Derivatives.svelte

```diff
@@ -1,76 +1,82 @@
+  export let currentSources = []
 
   onMount(() => {
-    if (currentSources.length) showDerivative = true;
-  });
+    if (currentSources.length) showDerivative = true
+  })
 
   async function handleAutoCompleteRequest(value) {
-    if (!value) return [];
+    if (!value) return []
 
-    return new FetchRails(`/code/${value}`).get({parameters: {headers: {"Accept": "application/json"}}, returnResponse: true}).then(async response => {
+    return new FetchRails(`/code/${ value }`).get({
+      returnResponse: true,
+      parameters: {
+        headers: { "Accept": "application/json" }
+      }
+    }).then(async response => {
       if (response.ok) {
-        const json = await response.json();
-        return json.map(post => postToResult(post));
+        const json = await response.json()
+        return json.map(post => postToResult(post))
       } else {
-        throw new Error(`${response.status} ${response.statusText}`);
+        throw new Error(`${ response.status } ${ response.statusText }`)
       }
     })
   }
 
   function postToResult(post) {
     return {
       label: post.code,
-      html: `<strong>${post.code.toUpperCase()}</strong> - ${post.title} by ${post.user.username}`
-    };
+      html: `<strong>${ post.code.toUpperCase() }</strong> - ${ post.title } by ${ post.user.username }`
+    }
   }
 </script>
 
-{#if false}
+
+
 <div class="form-group mt-1/4">
   <div class="switch-checkbox">
     <input
       id="show_derivative"
       class="switch-checkbox__input"
       autocomplete="off"
       type="checkbox"
-      bind:checked={showDerivative}
-    >
+      bind:checked={ showDerivative }>
+
     <label
       class="switch-checkbox__label"
-      for="show_derivative"
-    >
+      for="show_derivative">
       This code is a fork/uses other codes
     </label>
   </div>
 
-  {#if showDerivative}
+  { #if showDerivative }
     <div class="form-group mt-1/4">
-      <div class="form-hint text-left">
-        Enter the import code(s) which your mode uses. You can enter up to {maxCodes} codes.
+      <div class="form-hint mt-1/4 mb-1/4 text-left">
+        Enter the import code(s) which your mode uses. You can enter up to { maxCodes } codes.
+        
         <br />
         <strong>Separate import codes with a comma (<code>,</code>).</strong>
       </div>
 
       <Tags
         name="derivatives"
         placeholder="CODE1,CODE2,etc."
-        fillValues={currentSources}
-        hidden={!showDerivative}
-        allowSpace={false}
-        onlyAlphanumeric={true}
-        onlyCaps={true}
-        tagLimit={maxCodes}
-        useAutoComplete={true}
-        fetchAutoCompleteValues={handleAutoCompleteRequest}
-      />
+        fillValues={ currentSources }
+        hidden={ !showDerivative }
+        allowSpace={ false }
+        onlyAlphanumeric={ true }
+        onlyCaps={ true }
+        tagLimit={ maxCodes }
+        useAutoComplete={ true }
+        fetchAutoCompleteValues={handleAutoCompleteRequest } />
```

Comment:
I think a space got missed here. Shouldn't this be
```
fetchAutoCompleteValues={ handleAutoCompleteRequest } />
```
More importantly, did the linter not catch this in Actions?

Reply:
Unfortunately the linter only checks the <script> blocks.